README.horizontal-scaling.md

```diff
@@ -0,0 +1,160 @@
+## Horizontal worker scaling
+
+Automated horizontal worker scaling, when enabled, is run via a cron job on the ganglia
+monitoring node (monitoring-master1). It uses [the mo-scaler](https://github.com/harvard-dce/mo-scaler)
+python software to increase and decrease the number of worker nodes for a cluster
+based on one of two strategies.
+
+The chef recipe `mh-opsworks-recipes::install-moscaler` installs the
+necessary python requirements, the git repository and configures a `.env` file
+automatically with the necessary credentials and other environment variables from 
+your cluster config settings. If you change any of the scaling
+configuration settings (detailed below) you should re-rerun this recipe.
```

Comment:
Does it make sense to call it "update-moscaler" vs "install-moscaler" to reflect that it can be run many times and should be run after a config update? Along the lines of update config?